test/fixers/test_black_fixer_callback.vader

```diff
@@ -0,0 +1,39 @@
+Before:
```

Comment:
This test looks fine to me. What you need to do for Windows is create another empty executable file in the `Scripts` directory, in addition to the file for Linux in the `bin` directory, and I think you need to name it `.exe` for it to work. Have a look at some of the other dummy files for Windows testing.